e2e/portal/tests/__screenshots__/ExportRegistrations/ExportRegistrationsDataChanges.spec.ts/exported-xlsx.csv

```diff
@@ -1,2 +1,2 @@
-paId,referenceId,changedBy,type,newValue,oldValue,reason
-1,44e62864557597e0d,admin@example.org,registrationStatusChange,included,new,default reason
\ No newline at end of file
+id,programId,registrationId,registrationProgramId,type,fieldChanged,oldValue,newValue,reason,duplicateWithRegistrationId,duplicateWithRegistrationProgramId,username,userId
+1,2,2,1,registrationStatusChange,status,,new,,,,admin@example.org,1
```

Comment:
hmm ok, I just realized in fixing this final test that this export now comes out likes this, which is not really user-friendly.. Which would bring back the need for some xlsx mapper after all (or at least a 'select' on the columns). But maybe let's discuss the whole thing first, before throwing things upside down again.